docs/add-a-sut.md

```diff
@@ -0,0 +1,113 @@
+
+Adding a new SUT to ModelBench can be done in a number of ways, but here is an example of the easiest. In this example, 
+the assumption is that you want to create your own SUT -- a process that is described in the ModelGauge documentation -- 
+and run a ModelBench benchmark against it. Here are the steps to do so.
+
+## 1. Install ModelBench
+
+First things first, install ModelBench and make sure it works. For installation instructions, look [here](https://github.com/mlcommons/modelbench?tab=readme-ov-file#modelbench) 
+and [here, specifically](https://github.com/mlcommons/modelbench?tab=readme-ov-file#trying-it-out) for trying it out 
+instructions.
+
+This tutorial will assume you've installed ModelBench using PyPi
+
+## 2. Create your SUT plugin
+
+For the purposes of this tutorial, please refer to the **[Creating a basic SUT](https://github.com/mlcommons/modelgauge/blob/main/docs/tutorial_suts.md#creating-a-basic-sut)** 
+tutorial in the ModelGauge repository. Ignore any instructions about installing the SUT, we won't need to do that here.
+
+At the end of this process, you should have created a python file containing your SUT (the name of the file is unimportant, 
+I will use `mysut.py` for this tutorial example) containing a line that will register the SUT with ModelGauge like the 
+following:
+
+```python
+SUTS.register(DemoYesNoSUT, "demo_yes_no")
+```
+
+If you don't want to follow the instructions on creating a basic SUT right now, don't worry about it! Use this:
+
+```python
+from modelgauge.prompt import ChatPrompt, TextPrompt
+from modelgauge.prompt_formatting import format_chat
+from modelgauge.sut import PromptResponseSUT, SUTCompletion, SUTResponse
+from modelgauge.sut_capabilities import AcceptsChatPrompt, AcceptsTextPrompt
+from modelgauge.sut_decorator import modelgauge_sut
+from modelgauge.sut_registry import SUTS
+from pydantic import BaseModel
+
+class DemoYesNoRequest(BaseModel):
+    text: str
+
+class DemoYesNoResponse(BaseModel):
+    number_of_words: int
+    text: str
+
+@modelgauge_sut(capabilities=[AcceptsTextPrompt, AcceptsChatPrompt])
+class DemoYesNoSUT(PromptResponseSUT[DemoYesNoRequest, DemoYesNoResponse]):
+  def translate_text_prompt(self, prompt: TextPrompt) -> DemoYesNoRequest:
+    return DemoYesNoRequest(text=prompt.text)
+
+  def translate_chat_prompt(self, prompt: ChatPrompt) -> DemoYesNoRequest:
+    return DemoYesNoRequest(text=format_chat(prompt))
+
+  def evaluate(self, request: DemoYesNoRequest) -> DemoYesNoResponse:
+    # Return Yes if the input is an even number of words
+    number_of_words = len(request.text.split())
+    answer = "Yes" if number_of_words % 2 == 0 else "No"
+    return DemoYesNoResponse(number_of_words=number_of_words, text=answer)
+
+  def translate_response(
+    self, request: DemoYesNoRequest, response: DemoYesNoResponse
+) -> SUTResponse:
+    return SUTResponse(completions=[SUTCompletion(text=response.text)])
+
+SUTS.register(DemoYesNoSUT, "demo_yes_no")
```

Comment:
Could we rename this? Just to account for the small possibility that the user might already have the modelgauge-demo plugin installed.